scope_registry_test.go

```diff
@@ -140,3 +140,29 @@ func TestNewTestStatsReporterManyScopes(t *testing.T) {
 		wantHistograms, r.counters[histogramCardinalityName].val,
 	)
 }
+
+func TestForEachScopeConcurrent(t *testing.T) {
+	root := newRootScope(ScopeOptions{Prefix: "", Tags: nil}, 0)
+	go func() {
```

Comment:
Doesn't this leak goroutines during the test?

Reply:
good catch, fixing...